instrumentation/github.com/astaxie/beego/otelbeego/go.mod

```diff
@@ -5,9 +5,10 @@ go 1.14
 require (
 	github.com/astaxie/beego v1.12.2
 	github.com/stretchr/testify v1.6.1
-	go.opentelemetry.io/contrib v0.11.0
-	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.11.0
-	go.opentelemetry.io/otel v0.11.0
+	go.opentelemetry.io/contrib v0.12.0
+	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.12.0
+	go.opentelemetry.io/contrib/propagators v0.0.0-20200924185937-b313ddb2989e
```

Comment:
Good catch! Go was being helpful and inserting that reference since there weren't any tags at all for that module.  It also obscured some bad replace directives. I think I've caught all of them, but please give it a look.